src/components/prometheusmetrics.ts

```diff
@@ -135,7 +135,7 @@ export class PrometheusMetrics {
             * the lesser of two evils.
             */
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        const matrixClientPrototype = (MatrixClient as any).prototype;
+        const matrixClientPrototype = (JsSdk as any).MatrixClient.prototype;
```

Comment:
Is `as any` still required because `matrix-js-sdk` does not ship with types?

Reply:
It doesn't ship with MatrixClient exported as a type, despite being there. This is a ugly hack, so it causes uglyness :(